GridLock/GridLockWindow.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  GridLockWindow.swift
+//  GridLock
+//
+//  Created by Matt Wilson on 20/03/2019.
+//  Copyright © 2019 Instil. All rights reserved.
+//
+
+import Foundation
+
+class GridLockWindow: UIWindow {
+    
+    let gridLockViewController: GridLockViewController
+    var nextUIResponder: UIResponder?
```

Comment:
Use camel case i.e. `nextUiResponder `

Reply:
This is against the swift style e.g. `UIResponder`